corehq/apps/hqmedia/templates/hqmedia/partials/hqm_upload_form.html

```diff
@@ -1,29 +0,0 @@
-{% load i18n %}
-<fieldset class="hqm-sharing hide">
```

Comment:
This is all hidden by default due to the `hide` class. The way to show it is via the "share media" checkbox (see [here](https://github.com/dimagi/MediaUploader/blob/1038b486ecfca4251bb5c85bc427a92df35cf8d7/hqmedia.upload_controller.js#L211-L214)), but HQ doesn't have that checkbox. It did a long time ago (see [here](https://github.com/dimagi/commcare-hq/commit/eb3f82f2df7be1e07660b10f8b5b99f8147dbc0b#diff-24d4c57eafa007ed37b5fad665493bda8ebdeec7a33470d87efed3928560ff1cR9)) but it was removed at some point.